### PR TITLE
ICMSLST-2953 - Import Licence Data extract report ordering

### PR DESCRIPTION
**Pull request by marcuspp**

### CODE REVIEW

File: documentation/managing-dependencies.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ lock file, such as removed dependencies or packages manually installed using pip
 2. Run `make update-requirements` to regenerate all requirements
   Note: This will also update indirect dependencies.
 
-3. Run `pip-sync` to install the new locked dependencies locally. (You can also use
+3. Run `pip-sync requirements-dev.txt` to install the new locked dependencies locally. (You can also use
```
Comment (marcuspp) [inline documentation/managing-dependencies.md:41]: Unrelated doc fix

**Review by MattHolmes123 — APPROVED** [review]
